Source/Graphic/HCContourCurve.h

```diff
@@ -0,0 +1,69 @@
+//
+//  HCContourCurve.h
+//  HollowCore
+//
+//  Created by Matt Stoker on 2/11/20.
+//  Copyright © 2020 HollowCore. All rights reserved.
+//
+
+#ifndef HCContourCurve_h
+#define HCContourCurve_h
+
+#include "../Geometry/HCPoint.h"
+
+//----------------------------------------------------------------------------------------------------------------------------------
+// MARK: - Type
+//----------------------------------------------------------------------------------------------------------------------------------
+typedef struct HCContourCurve {
+    HCPoint c0;
+    HCPoint c1;
+    HCPoint p;
+} HCContourCurve;
+
+//----------------------------------------------------------------------------------------------------------------------------------
+// MARK: - Contour Curve Special Values
+//----------------------------------------------------------------------------------------------------------------------------------
+#define HCContourCurveInvalidStatic {.c0 = HCPointInvalidStatic, .c1 = HCPointInvalidStatic, .p = HCPointInvalidStatic}
+#define HCContourCurveZeroStatic    {.c0 = HCPointInvalidStatic, .c1 = HCPointInvalidStatic, .p = HCPointZeroStatic}
```

Comment:
We probably want a type case before the `Static` literal values here.

Reply:
Done.